src/main/java/frc/team2412/robot/commands/index/IndexIngestMotorOutCommand.java

```diff
@@ -0,0 +1,30 @@
+package frc.team2412.robot.commands.index;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.team2412.robot.subsystem.IndexSubsystem;
+
+public class IndexIngestMotorOutCommand extends CommandBase {
+    private final IndexSubsystem subsystem;
+
+    public IndexIngestMotorOutCommand(IndexSubsystem subsystem) {
+        this.subsystem = subsystem;
+        addRequirements(subsystem);
+    }
+
+    @Override
+    public void initialize() {
+        subsystem.ingestMotorOut();
+    }
+
+    @Override
+    public void execute() {
+        if (!subsystem.ingestSensorHasBallIn()) {
```

Comment:
Is the logic here correct?  Will it ever start if there is a ball in the feeder position and we want it out?